mupdf-sys/build.rs

```diff
@@ -110,11 +114,22 @@ fn build_libmupdf() {
         "HAVE_X11=no".to_owned(),
         "HAVE_GLUT=no".to_owned(),
         "HAVE_CURL=no".to_owned(),
+        // The parameter doesn't seem to have been implemented properly.
+        // - There's an error message about libmupdf.a.in not being found.
+        // - This will cause tests to fail under MYS2.
+        // "USE_ARGUMENT_FILE=yes".to_owned(),
         "verbose=yes".to_owned(),
     ];
 
-    for (feature, flag, make_flag, define) in CPU_FLAGS {
-        let contains = target_features.contains(feature);
+    let mut cpu_flags = DEFAULT_CPU_FLAGS.to_vec();
+    let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap_or_default();
+
+    if target_arch == "arm" {
```

Comment:
```suggestion
    if target_arch == "arm" || target_arch == "aarch64" {
```

also need to consider `aarch64`?

Reply:
Previously, I only had `aarch64` and the build was failing on GitHub Actions (OK locally on Mac OS x86_64). 

I will give it a try though.